packages/discord.js/src/structures/UncachedGuildMember.js

```diff
@@ -0,0 +1,332 @@
+'use strict';
+
+const Base = require('./Base');
+const TextBasedChannel = require('./interfaces/TextBasedChannel');
+const { GuildMemberFlagsBitField } = require('../util/GuildMemberFlagsBitField');
+const PermissionsBitField = require('../util/PermissionsBitField');
+
+/**
+ * Represents a member of a guild on Discord. Used in interactions from guilds that aren't cached.
+ * Backwards compatible with {@link APIGuildMember}.
+ * @implements {TextBasedChannel}
+ * @extends {Base}
+ */
+class UncachedGuildMember extends Base {
+  constructor(client, data, guildId) {
+    super(client);
+
+    /**
+     * The guild that this member is part of
+     * @type {Guild}
```

Comment:
Its a string, not a Guild. You can add in a guild getter

Reply:
oops will fix